### PR TITLE
Implemented selectors

### PR DESCRIPTION
**Pull request by JForsaken**
#27

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -14,5 +15,6 @@ export default {
   Link,
   navigate,
   normalizeRoutes,
+  selectors,
```
Comment (tomzmtl) [inline src/index.js]: I think we should expose all of them directly here, like `redux-form` does.

File: src/selectors.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default {
+  getRoute: (state) => state.router.route,
+  getRoutes: (state) => state.router.routes,
+  getRouteName: (state) => state.router.route.name,
+  getRoutePath: (state) => state.router.route.path,
+  getQuery: (state) => state.router.route.query,
+  getQueryParam: (state, name) => state.router.route.query[name],
```
Comment (tomzmtl) [inline src/selectors.js]: Should return `null` if the param name doesn't exist.
Also, all those should be unit tested.

**Comment by JForsaken** [comment]
@tomzmtl done

**Comment by JForsaken** [comment]
@tomzmtl done

**Pull request merged**